upload/system/storage/vendor/jwt/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "firebase/php-jwt",
```

Comment:
please update composer.json and composer.lock in the project root

Reply:
fixed it.